include/unifex/async_scope.hpp

```diff
@@ -170,6 +170,14 @@ struct async_scope {
     }
   }
 
+  template (typename Fun, typename Scheduler)
+    (requires callable<Fun> && scheduler<Scheduler>)
+  void spawn_with(Fun&& fun, Scheduler&& scheduler) {
```

Comment:
I'm not in love with the name `spawn_with`.
The `with` part is presumably referring to the "scheduler" but this is not something that is different from the sender-form of `spawn()`.

Maybe something like `spawn_call(fn, scheduler)` or `spawn_call_on(fn, scheduler)` would better indicate that this is for callables/invocables?

Reply:
What does the `_with` in `let_with` refer to? I thought it was referring to the lambda, which is why I used it here.

Reply:
Yes, let_with refers to the lambda. The meaning intended was more "let_with_the_value_from_the_lambda". let didn't itself refer to the function. Spawn feels like it does already refer to the function, though, so maybe that's why this seems different.